daffodil-test/src/test/resources/org/apache/daffodil/section13/text_number_props/TextNumberProps.tdml

```diff
@@ -4039,16 +4081,13 @@
       <tdml:documentPart type="text"><![CDATA[NOTANUMBER]]]></tdml:documentPart>
     </tdml:document>
 
-    <tdml:warnings>
-      <tdml:warning>DFDL property was ignored</tdml:warning>
-      <tdml:warning>textStandardNaNRep</tdml:warning>
-    </tdml:warnings>
+    <tdml:warnings />
```

Comment:
Can we just delete the empty tdml:warnings element without changing any semantics?

Reply:
The empty `tdml:warnings` explicitly says "there should be no warnings", whereas not having a `tdml:warnings` element just says "we don't care if there are warnings or not". The idea with changing it like this was that the test used to expect a warning saying this property was ignored, but the property is now used so I wanted to make sure it's not ignored and there's no warning. We do get an error mentioning "NaN" so that should be enough to make it clear the property is used, but I wanted to still keep the intention of the test regarding warnings.

Reply:
I wasn't aware that empty tdml:warnings meant that. Does that also work for empty tdml:validationErrors?

I think there is a JIRA ticket [about adding this feature for expecting/precluding warnings.](https://issues.apache.org/jira/browse/DAFFODIL-864) Perhaps this feature exists now.

Reply:
Looks like I was confused. It looks like we have that logic for `tdml:validationErrors` but not `tdml:warnings`. So you can ensure there are no validation errors, but can't do the same for warnings.

So this change does nothing at the moment.

Reply:
@stevedlawrence, you can leave the empty `tdml:warnings` since it might gain the expected semantics later on.

@mbeckerle and @mike-mcgann, this PR needs a second +1 from one of you.